### PR TITLE
MOSIP-32635 Corrected Readme for java21

### PR DESCRIPTION
**Pull request by kameshsr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +61,7 @@ Prerequisites:
 To deploy Commons services on Kubernetes cluster using Dockers refer to [Sandbox Deployment](https://docs.mosip.io/1.2.0/deployment/sandbox-deployment).
 
 ## Configuration
-Refer to the [configuration guide](docs/configuration.md).
+[Configuration](https://github.com/mosip/mosip-config/blob/develop/resident-default.properties) defined here.
```
Comment (dhanendra06) [inline README.md]: Add the respective configuration

**Review by dhanendra06 — CHANGES_REQUESTED** [review]
specify the required jar to be add in classpath.

File: README.md (hunk 1 above)
Comment (kameshsr) [reply README.md]: Done

**Comment by kameshsr** [comment]
> specify the required jar to be add in classpath.

Done.

File: README.md [hunk 2]
```diff
@@ -13,32 +13,79 @@ See [DB guide](db_scripts/README.md)
...
-## Test
+### Test
 Automated functional tests available in [Functional Tests repo](https://github.com/mosip/mosip-functional-tests).
 
-## APIs
+### APIs
 API documentation is available [here](https://mosip.github.io/documentation/).
 
-## License
+### License
```
Comment (dhanendra06) [inline README.md]: its heading please add ## on it for subheading use ###

Comment (kameshsr) [reply README.md]: Corrected

**Review by dhanendra06 — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Pull request merged**